wikidict/render.py

```diff
@@ -0,0 +1,335 @@
+import json
+import os
+from collections import defaultdict
+from itertools import chain
+from pathlib import Path
+from typing import Dict, List, Pattern
+
+from scripts.lang import (
+    definitions_to_ignore,
+    genre,
+    etyl_section,
+    head_sections,
+    pronunciation,
+    section_level,
+    section_sublevels,
+    section_patterns,
+    sections,
+    sublist_patterns,
+    words_to_keep,
+)
+from scripts.stubs import Definitions, SubDefinitions, Word, Words
+from scripts.utils import clean, process_templates
+
+
+import wikitextparser as wtp
+import wikitextparser._spans
+
+# As stated in wikitextparser._spans.parse_pm_pf_tl():
+#   If the byte_array passed to parse_to_spans contains n WikiLinks, then
+#   this function will be called n + 1 times. One time for the whole byte_array
+#   and n times for each of the n WikiLinks.
+#
+# We do not care about links, let's speed-up the all process by skipping the n times call.
+# Doing that is a ~30% optimization.
+wikitextparser._spans.WIKILINK_PARAM_FINDITER = lambda *_: ()
```

Comment:
Note: I changed this line compared to `get.py`
WIKILINK_FINDITER doesn't exist anymore in wtp.